modules/Book.js

```diff
@@ -0,0 +1,133 @@
+const bookApp = (bookStorageName) => {
+  let counter = 0;
+  let bookList = [];
+  const form = document.querySelector('form');
+  const contactUs = document.querySelector('.contactUs');
+  const booksSection = document.querySelector('#book-section');
+  const bookListElement = document.querySelector('#book-list');
+  const contBtn = document.querySelector('#contBtn');
+  const listBtn = document.querySelector('#listBtn');
+  const AddNewBtn = document.querySelector('#AddNewBtn');
+
+  const hideAll = () => {
+    if (!contactUs.classList.contains('hidden')) {
+      contactUs.classList.add('hidden');
+    }
+
+    if (!booksSection.classList.contains('hidden')) {
+      booksSection.classList.add('hidden');
+    }
+
+    if (!form.classList.contains('hidden')) {
+      form.classList.add('hidden');
+    }
+  };
+
+  hideAll();
+  booksSection.classList.toggle('hidden');
+
+  contBtn.addEventListener('click', () => {
+    hideAll();
+    contactUs.classList.toggle('hidden');
+  });
+
+  listBtn.addEventListener('click', () => {
+    hideAll();
+    booksSection.classList.toggle('hidden');
+  });
+
+  AddNewBtn.addEventListener('click', () => {
+    hideAll();
+    form.classList.toggle('hidden');
+  });
+
+  class Book {
+    constructor(title, author) {
+      this.title = title;
+      this.author = author;
+      this.newBookList = [];
+    }
+
+    saveData() {
+      this.newBookList = [];
+      bookList.forEach((book) => {
+        this.newBookList.push({ title: book.title, author: book.author });
+      });
+      localStorage.setItem(bookStorageName, JSON.stringify(this.newBookList));
+    }
```

Comment:
- [ ] 👍 Your `saveData` function works perfectly, and that's great! As this project specifically requires the use of ES6 syntax, I would recommend utilizing arrow functions instead of regular functions. This practice aligns with the ES6 standard and enhances the readability and maintainability of the code. Please consider updating your code to use arrow functions wherever appropriate. Keep up the good work, and happy coding! 🚀🎉
- [ ] Check all functions in all files and use arrow functions instead of regular functions👍